CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
Since this is a breaking change, this should be `3.0.0`

Reply:
Ok for me